src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,47 @@
+package racingcar;
+
+import exception.CustomException;
+
+public class Car {
+    private final String name;
+    private int position = 0;
+    public static final int MINIMUM_MOVE_VALUE = 4;
+    public static final String FORWARD_UNIT = "-";
+
+    public Car(String name) {
```

Comment:
상수화 좋습니다~ 자바 컨벤션 보시고 순서도 유의해보시고 접근제어자도 신경써보시면 좋을 것 같아요.
추가적으로 position 값이 기본으로 0이라면 생성자나 정적 팩토리 메서드를 별도로 만들어서 의미를 부여해주면 어떨까요?
-> 이펙티브 자바에서 정적 팩토리 메서드 부분 같이 읽어보시면 좋을 것 같습니다

private int position = 0; --> public static Car createDefault(String name) { return new Car(name, DEFAULT_MOVE_COUNT); } 이런식으로요 ㅎㅎ

```suggestion
public class Car {

    private static final int MINIMUM_MOVE_VALUE = 4;
    private static final String FORWARD_UNIT = "-";
    
    private final String name;
    private int position = 0;

    public Car(String name) {
```

Reply:
정적 팩토리 메서드로 생성 완료했습니다!
생성자가 명시적이지 않을 때
정적 팩토리 메서드를 사용하는것을 확인했습니다!
읽어야 할 책이 많겠군요,,ㅎㅎ